apps/cli/src/cli.ts

```diff
@@ -111,10 +116,17 @@ export async function cli(): Promise<void> {
 
   await logger.logHeader();
 
-  let goal: string | undefined = process.argv[2];
+  program
+    .option("-t, --timeout <number>")
+    .option("-g , --goal <string>")
```

Comment:
IMO, UX is better like this:
`yarn start "do something" -t 100`
instead of:
`yarn start -g "do something" -t 100`
Especially when there is no timeout:
`yarn start "do something"`
vs
`yarn start -g "do something"`

Reply:
yah either way is good for me. i think we still need the `-t` because that way use can do:
```
yarn start -t 200
```
without needing to define a goal at the start